Source/OxyPlot.SkiaSharp/SkiaExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SkiaExtensions.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.SkiaSharp
+{
+    using global::SkiaSharp;
+
+    /// <summary>
+    /// Provides extension methods for conversion between SkiaSharp and oxyplot objects.
+    /// </summary>
+    public static class SkiaExtensions
+    {
+        /// <summary>
+        /// Converts a <see cref="OxyColor"/> to a <see cref="SKColor"/>;
+        /// </summary>
+        /// <param name="color">The <see cref="OxyColor"/>.</param>
+        /// <returns>The <see cref="SKColor"/>.</returns>
+        public static OxyColor ToOxyColor(this SKColor color)
+        {
+            return OxyColor.FromArgb(color.Alpha, color.Red, color.Green, color.Blue);
+        }
+
+        /// <summary>
+        /// Converts a <see cref="SKColor"/> to a <see cref="OxyColor"/>;
+        /// </summary>
+        /// <param name="color">The <see cref="SKColor"/>.</param>
+        /// <returns>The <see cref="OxyColor"/>.</returns>
+        public static SKColor ToSKColor(this OxyColor color)
+        {
+            if (color.IsAutomatic() || color.IsUndefined())
+            {
+                return SKColors.Empty;
+            }
```

Comment:
Would it be better to preserve the exact values so that these methods can round-trip?

Reply:
My reasoning behind this was that SkiaSharp does not know that the ARGB [0,0,0,1] value is somehow 'special' to us.
But thinking about it, it doesn't really matter... For rendering it should make no difference and the round-trip argument is valid.